libs/libcommon/src/libcommon/config.py

```diff
@@ -108,7 +108,8 @@ class ProcessingGraphConfig:
             "/parquet": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
             "/dataset-info": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
             "/split-names-from-dataset-info": {"input_type": "config", "requires": "/dataset-info"},
-            "/sizes": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
```

Comment:
Could we keep for a while the "/sizes" processing step? so then, once the data is processed in prod, we will switch to it when we validate everything is correct. In a new PR we could: delete the processing step and make the "/sizes" endpoint use "/dataset-size" and remove the cached data for "/sizes" and start using the new one.